code/app/Http/Controllers/TodosController.php

```diff
@@ -67,4 +67,15 @@ public function apiresponseget()
     {
         return view('yourtasksresp');
     }
+
+    public function SPAtasks(Request $request)
+    {
+        $tasks =  Todo::readTasks();
```

Comment:
zamiast wołać `readTasks()` posłuż się relacjami z eloquenta: https://laravel.com/docs/5.8/eloquent-relationships#one-to-many

ostatecznie wystarczy że zawołasz:
```
$user->tasks;
```